### DISCUSSION TITLE
ColorPicker onChange 参数返回的值修改了，却没有任何修正

### DISCUSSION THREAD

**Discussion by one-pieces** [ant-design/ant-design#51868] [General]
最近把 antd 从 5.13.3 升到 5.22.2，结果 ColorPicker onChange 函数的第二个参数 css，从原本的 hex 变成了 rgb，导致出现 bug 了。
<img width="897" alt="image" src="https://github.com/user-attachments/assets/99f67f4e-b120-4812-9c04-b847420ada88">

而这个改动官网根本没有任何修正。

<img width="1715" alt="image" src="https://github.com/user-attachments/assets/c6522c5f-d462-47f0-a8f7-6ae4bbc25330">

为什么要改这个值的格式？

**Comment by zombieJ** [comment]
底层的颜色库改了，我测一下看看原本的逻辑。

**Reply by zombieJ** [reply]
https://stackblitz.com/edit/react-nwwkdx?file=demo.tsx,package.json

1. `onChange` 返回的 `cssStr` 的确是从 `hex` 变成了 `rgb` 的格式。如果你需要 hex 格式，可以通过第一个参数的 color 对象获取。
2. 这里的确是疏忽，但是文档中并没有保证返回的格式类型。你圈的那个是面板的展示的 `format`，和 `onChange` 返回的参数格式是没有关系的。

**Reply by one-pieces** [reply]
感谢回复，已经用第一个方式修复了
